pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -151,9 +179,67 @@ def cli(  # noqa: C901
     parser.add_argument('--no-stream', action='store_true', help='Disable streaming from the model')
     parser.add_argument('--version', action='store_true', help='Show version and exit')
 
+    if prog_name == 'clai':
```

Comment:
Why do we need to check this?

Reply:
because there's also the `pai` program, right? if I remove this though then that probably would make it so users can also run `pai web`

I just assumed I should restrict it. what do you think?

Reply:
that's probably the case, I haven't found any external references to it. will remove the program check